notebooks/community/model_garden/model_garden_video_object_tracking_bytetrack.ipynb

```diff
@@ -0,0 +1,679 @@
+{
```

Comment:
<p>I would drop this code cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2098/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>